vm_args.c

```diff
@@ -494,7 +510,7 @@ setup_parameters_complex(rb_execution_context_t * const ec, const rb_iseq_t * co
 	else {
 	    args->kw_argv = NULL;
 	    given_argc = args_kw_argv_to_hash(args);
-	    kw_flag |= VM_CALL_KW_SPLAT;
+            kw_flag |= VM_CALL_KW_SPLAT | VM_CALL_KW_SPLAT_MUT;
```

Comment:
Whitespace 👮

Reply:
New and modified lines do not use tabs (this has been the policy since some time last year), so the whitespace changes for changed lines are expected.